index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
```

Comment:
why was this necessary to move to a diff file? Instead, why not have an index.html with a nav, and the cabinets.html should be one of the items in the nav